services/clusters-keeper/.env-devel

```diff
@@ -0,0 +1,27 @@
+EC2_ACCESS_KEY_ID=XXXXXXXXXX
```

Comment:
TIP: instead of having an `.env-devel` another alternative (perhaps a bit more sustainable) is to add in the sevice CLI the capability to generate automatically an `.env` from a template. As example look at 

https://github.com/ITISFoundation/osparc-simcore/blob/95c2073846c109e71c3421197376762ff5f84af7/services/invitations/src/simcore_service_invitations/cli.py#L75-L107

This can even be generalized and add it to https://github.com/ITISFoundation/osparc-simcore/blob/56f5ea1aa64c79a2c672c584ece1cff7130c8a03/packages/settings-library/src/settings_library/utils_cli.py

Reply:
will think about it later. not necessary for now.